package.json

```diff
@@ -111,34 +110,34 @@
     "lodash": "^4.17.21",
     "markdown-it": "^14.1.0",
     "markdown-to-txt": "^2.0.1",
-    "mercurius": "^14.1.0",
-    "mercurius-cache": "^6.0.1",
-    "mercurius-upload": "^6.0.1",
+    "mercurius": "^15.1.0",
+    "mercurius-cache": "^7.0.2",
+    "mercurius-upload": "^7.0.0",
```

Comment:
Looks like tests fail due to https://github.com/mercurius-js/mercurius-upload/pull/24#issuecomment-2442001808, hopefully released soon

Reply:
patched it now because looks like there are multiple issues https://github.com/mercurius-js/mercurius-upload/issues/13#issuecomment-2443853365